File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    s = s.lower()
    cnt_p, cnt_y = 0, 0
    
    for k in s:
        if k == 'p':
            cnt_p += 1
        elif k == 'y':
            cnt_y += 1
    if cnt_p != cnt_y:
        answer = False
    return answer

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    let answer = true;

    const lowerString = s.toLowerCase();
    
    const countP = lowerString.match(/p/g)?.length
    const countY = lowerString.match(/y/g)?.length
    
    if (countP !== countY) return answer = false

    return answer;
}